internal/pkg/plugin/plugin.go

```diff
@@ -27,10 +27,16 @@ type pluginInfo struct {
 
 // SearchPath goes through the files in the user's $PATH and checks if they are plugins
 func SearchPath(cfg *v1.Config) map[string][]string {
-	log.CliLogger.Debugf("Searching `$PATH` for plugins. Plugins can be disabled in %s.\n", cfg.GetFilename())
+	log.CliLogger.Debugf("Searching `$PATH` and `~/.confluent/plugins` for plugins. Plugins can be disabled in %s.\n", cfg.GetFilename())
```

Comment:
Does Windows use `~` for `$HOME`?

Reply:
I don't think so, actually. I'll have to do some Windows testing tomorrow.

Reply:
Switched it to `%USERPROFILE%` which is Windows' version of `$HOME`, and also fixed the directories in `install` and `search` to use `os.UserHomeProfile` instead of `$HOME`, which didn't work on Windows.